glyph/src/ttfp.rs

```diff
@@ -203,6 +203,11 @@ macro_rules! impl_font {
                 self.0.as_face_ref().line_gap().into()
             }
 
+            #[inline]
+            fn italic_angle(&self) -> f32 {
+                self.0.as_face_ref().italic_angle().into()
```

Comment:
I think no conversion required here as upstream uses the same type.

```suggestion
                self.0.as_face_ref().italic_angle()
```